dev/create-release/release-build.sh

```diff
@@ -314,21 +325,23 @@ if [[ "$1" == "package" ]]; then
 
   rm -rf spark-$SPARK_VERSION-bin-*/
 
-  if ! is_dry_run; then
-    svn co --depth=empty $RELEASE_STAGING_LOCATION svn-spark
-    rm -rf "svn-spark/${DEST_DIR_NAME}-bin"
-    mkdir -p "svn-spark/${DEST_DIR_NAME}-bin"
-
-    echo "Copying release tarballs"
-    cp spark-* "svn-spark/${DEST_DIR_NAME}-bin/"
-    cp pyspark-* "svn-spark/${DEST_DIR_NAME}-bin/"
-    cp SparkR_* "svn-spark/${DEST_DIR_NAME}-bin/"
-    svn add "svn-spark/${DEST_DIR_NAME}-bin"
-
-    cd svn-spark
-    svn ci --username $ASF_USERNAME --password "$ASF_PASSWORD" -m"Apache Spark $SPARK_PACKAGE_VERSION" --no-auth-cache
-    cd ..
-    rm -rf svn-spark
+  if [ -z "$AMPLAB_JENKINS" ]; then
+    if ! is_dry_run; then
```

Comment:
maybe we should just set dry-run at jenkin build, then we don't need to change this part.

Reply:
i think this way is much more "clear" as to show what's not being run.